software/openvisualizer/bin/openVisualizerApp/web_files/templates/topology.tmpl

```diff
@@ -591,31 +594,40 @@
                         console.log("ERROR: could not GET topology data");
                     })
                     .always(function() {
-                        
+
                         // close info window
                         connectioninfo.close();
-                        
+
                         // resume automated update
                         clearMapLock();
                     });
                 })
             }
-            
+
             //=============== main ============================================
-            
-            google.maps.event.addDomListener(window, 'load', initialize);  
+
+            google.maps.event.addDomListener(window, 'load', initialize);
         </script>
 	</head>
 	<body>
 		<div id="wrapper">
 			%include navbar.tmpl ovVersion=ovVersion
-			
+
 			<div id="page-wrapper">
 	            <div class="row">
 	                <div class="col-lg-12">
 	                    <h1 class="page-header">Topology</h1>
 	                </div>
 	            </div>
+                <div id= "topology-options">
+                    <div class="row">
+                        <div class="col-lg-12">
+                            <h4>Topology options</h3>
+
+                            <a href="topology/download"><button id="toggle_btn" type="button" class="btn btn-default btn-xs">Save topology</button></a>
```

Comment:
please replace the id of the button from `toggle_btn` to `topology_download_btn`